src/Compilers/Lua/Test/Portable/Scoping/FindScopeTests.cs

```diff
@@ -32,5 +32,15 @@ public void FindScope_OnRootElement_ReturnsRootScope()
 
             Assert.Equal(compilationUnitScope, printExpressionScope);
         }
+
+        [Fact]
+        [Trait("Category", "Script/FindScope")]
+        public void FindScope_AnonymousFunctionIsParsed()
```

Comment:
Missing a `WorkItem` attribute:
```suggestion
        [WorkItem(106, "https://github.com/LorettaDevs/Loretta/issues/106")]
        public void FindScope_AnonymousFunctionIsParsed()
```